components/identifier-generator/front/src/feature/components/FamiliesSelector.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {FC} from 'react';
```

Comment:
```suggestion
import React, {FC, useState, useEffect} from 'react';
```
Why not import useState & useEffect methods, like in _**useGetFamilies.ts**_ file ?